src/views/forms/FormTransactionBase/FormTransactionBase.ts

```diff
@@ -272,16 +306,191 @@ export class FormTransactionBase extends Vue {
         isMultisig: this.isMultisigMode(),
       })
     }
-
-    // - add transactions to stage (to be signed)
-    await Promise.all(
-      transactions.map(async (transaction) => {
-        await this.$store.dispatch('account/ADD_STAGED_TRANSACTION', transaction)
-      }),
-    )
-
-    // - open signature modal
-    this.onShowConfirmationModal()
+    const options = this.$store.getters['account/stageOptions']
+    //check isLedger account
+    if (this.currentAccount.type == AccountType.fromDescriptor('Ledger')) {
```

Comment:
Separate how a transaction is signed from how it is announced and validated to simplify code. 

Please have a look at 
https://github.com/nemfoundation/symbol-desktop-wallet/pull/368/